### PR TITLE
feat(custome loader):nprogress added with custom loader

### LINKED ISSUE #75: Create a custom loading page design
**Is your feature request related to a problem? Please describe.**
When the user changes the page, then the screen freezes until the new page is loaded

**Describe how the solution should behave**
Make a loading page to display whenever the user changes the page

**Screenshots of the design/if any**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by NooraYaqub44**

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -22,16 +28,39 @@ const App = ({ Component, pageProps }) => {
       user ? dispatch(setIsLoggedIn(true)) : dispatch(setIsLoggedIn(false));
     });
   }, [dispatch]);
+
+  Router.events.on("routeChangeStart", (url) => {
+    console.log("Route is changing... ");
```
Comment (MuhannedNoman) [inline pages/_app.js]: Let's remove the console logs.

**Review by MuhannedNoman — APPROVED** [review]
LGTM, just a small comment.

**Review by HaiderAltufaily — APPROVED** [review]
LGTM, Nice work

File: pages/_app.js (hunk 1 above)
Comment (NooraYaqub44) [reply pages/_app.js]: okay

**Review by aryankarim — APPROVED** [review]
Great Work! Let me know if you still have issues with the conflicts.

**Pull request merged**